src/amici_interface_matlabo2.cpp

```diff
@@ -0,0 +1 @@
+#include "amici_interface_matlab.cpp"
```

Comment:
this is necessary to have debug symbols for both first and secondorder compilation, otherwise object file will be overwritten.